### ISSUE TITLE
How Can I set starting point from zero on xAxis (string value)?

### ISSUE THREAD

**Issue by avestaHarsh** [recharts/recharts#1772]
![rechart](https://user-images.githubusercontent.com/16647220/59325213-c9b2d400-8cff-11e9-90e0-e396903e7c4e.png)

I want to start xAxis record from zero, I have tried with domain on xAxis but unforunately it's not working, I think domain is only work on type number, but in my case type is string.

![Code1](https://user-images.githubusercontent.com/16647220/59325398-860c9a00-8d00-11e9-8efb-eff20cec84d2.png)

Can you please guide me ?

**Comment by sanehab** [comment]
I think that this is a problem of the data, the data does not have any value whose time is zero;

**Comment by avestaHarsh** [comment]
@sanehab , yes data does not have any value whoes time is zero, but I am not getting why dataMin in domain not working for string value i.e. time on xAxis.

**Comment by chiomadans1759** [comment]
@sanehab Has anyone found the answer to this. I sent a data that started from 0 but it still gave the left margin. 
<img width="590" alt="Screenshot 2020-09-22 at 3 38 21 PM" src="https://user-images.githubusercontent.com/26308412/93897183-aee45300-fce9-11ea-80ab-13cbce8b745b.png">
